bgpd/bgp_mplsvpn.c

```diff
@@ -1466,11 +1466,9 @@ static int bgp_mplsvpn_get_label_per_nexthop_cb(mpls_label_t label,
 	if (old_label == blnc->label)
 		return 0;
 
-	if (blnc->label != MPLS_INVALID_LABEL) {
-		bgp_zebra_send_nexthop_label(
-			ZEBRA_MPLS_LABELS_ADD, blnc->label, blnc->nh->ifindex,
-			blnc->nh->vrf_id, ZEBRA_LSP_BGP, &blnc->nexthop);
-	}
```

Comment:
do not remove what you added a few commits ago

Reply:
I think this was because of the braces.
now it should be okay. please review